tools/protoschema/BUILD

```diff
@@ -0,0 +1,28 @@
+load("@rules_python//python:defs.bzl", "py_binary")
+load("@protoschema_pip3//:requirements.bzl", "requirement")
+load("//bazel:envoy_build_system.bzl", "envoy_package", "envoy_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_proto_library(
+    name = "manifest_proto",
+    srcs = ["manifest.proto"],
+)
+
+py_binary(
+    name = "protoschema",
+    srcs = ["protoschema.py"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":manifest_proto_py_proto",
+        "//tools/api_proto_plugin",
+        "//tools/config_validation:validate_fragment",
+        "@com_envoyproxy_protoc_gen_validate//validate:validate_py",
+        "@com_github_chrusty_protoc_gen_jsonschema",
```

Comment:
adding this fails - im not sure how to include the lib so its visible to `api_proto_plugin_impl`

Reply:
seems like one problem im hitting is because `protoc_gen_jsonschema` doesnt have a bazel `BUILD` file 

im wondering if i need to "inject" a BUILD file as suggested here https://grpc.io/blog/bazel-rules-protobuf/#143-workspace-rules-that-accept-a-build-file-as-an-argument, or whether there is an easier way to include the compiled binary

also, not sure if/how its possible to use `rules_proto` - i had a look at it - but im struggling to figure out how it can be used with `protoc-gen-jsonschema`

Reply:
A few things:
1. You probably do need to inject a BUILD file (or upstream one) to turn protoc-gen-jsonschema into a `go_binary`.
2. I think the interesting thing from api_proto_plugin isn't the Python framework, which doesn't really apply here, but the .bzl, i.e. tools/api_proto_plugin/plugin.bzl, which gives essentially a way to invoke some protoc plugin as an aspect.
3. For `rules_proto`, I think you want to figure out how to use `proto_plugin` based on how some of the language examples work in the repo, e.g. `cpp_proto_compile`. If we go this route, we could either do an aspect like in (2), or modify `api_proto_library` to also externalize a jsonscheme target for each proto library.

Reply:
im gonna have to get a better handle on how these components work independently of envoy repo - ill update once i have a vague clue what im doing...